cmd/version/version.go

```diff
@@ -1,26 +1,24 @@
 package version
 
 import (
-	"fmt"
+	"github.com/edgexfoundry-holding/edgex-cli/pkg/cmd/version"
 
-	client "github.com/edgexfoundry-holding/edgex-cli/pkg"
 	"github.com/spf13/cobra"
 )
 
-var version string
-
 // NewCommand returns the version command
 func NewCommand() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "version",
 		Short: "Version command",
 		Long:  `Outputs the current version of edgex-cli.`,
```

Comment:
The documentation does not match.
s/edgex-cli/edgex-cli and edgex-go

Reply:
I have changed the Long message to : 
`Outputs the current versions of EdgeX CLI and EdgeX Foundry`

And in addition, I have changed the banner to - "EdgeX-CLI"
and the version of the CLI is immediately printed under the banner